sdk/purview/purview-workflow-rest/samples-dev/approveApprovalTaskSample.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I wonder if you could add some complex scenario sample here

Reply:
Instead just simple http calls sample.

Reply:
Thanks for your comment, the reason I just use simple calls here is Workflow is a straightforward service, each operation could be done via a simple http call, so I was thinking that these samples is eough for user's reference.